app/models/schemas/common_metadata.rb

```diff
@@ -93,6 +98,10 @@ module CommonMetadata
         index.as :stored_searchable, :facetable
       end
 
+      property :abstract, predicate: ::RDF::Vocab::DC.abstract, multiple: true do |index|
```

Comment:
Question - What's the change you are doing with the `abstract`?

We're you moving the `abstract` from here?
https://github.com/nulib/donut/blob/1ec61cb631c719db4622913961dfb479a350e617/app/models/schemas/workflow.rb

Reply:
no it shouldn't be touched in this pr, i think it's an errant copy/paste. i'll remove it and change stuff to the singular